src/test/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/adjudications/AdjudicationsHearingResultsResourceIntTest.kt

```diff
@@ -654,7 +654,7 @@ class AdjudicationsHearingResultsResourceIntTest : IntegrationTestBase() {
           .exchange()
           .expectStatus().isNotFound
           .expectBody()
-          .jsonPath("developerMessage").isEqualTo("Hearing Result not found. Hearing Id: 123, result sequence: 1")
+          .jsonPath("developerMessage").isEqualTo("Hearing Result not found. Hearing Id: 123, charge sequence: 1")
```

Comment:
was expecting incorrect message since this is by charge sequence not result sequence